R/session.R

```diff
@@ -72,8 +72,15 @@ session_info <- function(approved_folder_pkgs = NULL, approved_url_pkgs = NULL,
 
 python_package_info <- function(json) {
 
+  json <- jsonlite::fromJSON(json)
+
+  if (!length(json)) {
+    return(structure(list(Package = character(0), Version = character(0),
```

Comment:
here, we could use the tibble package if we want the tbl class

Reply:
you are right. at the moment we still have tibble as an import, so I might as well use it.